argus.js

```
@@ -3,21 +3,33 @@ const MESSAGE_ARGUS_TOKEN_REQUEST = "argus-token-request";
 const MESSAGE_ARGUS_TOKEN_RESPONSE = "argus-token-response";
 
 const ArgusJS = function(token, mdrUrl) {
-    let currentToken = token;
+    let auth
```

can you add error messaging 
if a token is null / undefinted or without .acesss it will fail silently. and document what the expected format to use in the readme

---

Good idea I'll put a check in. The ArgusJS constructor isn't actually exposed (just initArgusJS) so token will only come from a MESSAGE_ARGUS_TOKEN_RESPONSE message event